src/controllers/trading.stop.controller.ts

```diff
@@ -0,0 +1,16 @@
+import { Request, Response } from "express";
+import { Templates } from "../types/template.paths";
+import { urlUtils } from "../utils/url";
+import { TRADING_STATUS_PATH } from "../types/page.urls";
+
+export const get = (req: Request, res: Response) => {
+
+  const companyNumber = urlUtils.getCompanyNumberFromRequestParams(req);
+  const transactionId = urlUtils.getTransactionIdFromRequestParams(req);
+  const submissionId = urlUtils.getSubmissionIdFromRequestParams(req);
+  return res.render(Templates.TRADING_STOP, {
```

Comment:
there's a more concise way to get the url - you can use this
return res.render(Templates.TRADING_STOP, {
      backLinkUrl: urlUtils.getUrlToPath(TRADING_STATUS_PATH, req)
    });